Library/Homebrew/cmd/audit.rb

```diff
@@ -1234,4 +1224,9 @@ def audit_urls
   def problem(text)
     @problems << text
   end
+
+  if ARGV.switch? "D"
+    Homebrew.inject_dump_stats!(FormulaAuditor, /^audit_*/)
+  end
 end
+
```

Comment:
No worries. I guess I (or we) should give you some time to catch up with all the feedback. :smiley_cat: